cmake/CompilerHelper.cmake

```diff
@@ -54,9 +54,3 @@ else()
   set(USING_PGI_COMPILER_TRUE "#")
   set(USING_PGI_COMPILER_FALSE "")
 endif()
-
-if(CMAKE_CXX_COMPILER_ID MATCHES "PGI")
-  # CMake adds strict standard complaint PGI flag "-A" which breaks compilation of old codes (e.g.
-  # spdlog, fmt)
-  set(CMAKE_CXX14_STANDARD_COMPILE_OPTION --c++14)
-endif()
```

Comment:
In the CoreNEURON PR I added: https://github.com/BlueBrain/CoreNeuron/blob/4582adaa3ea6b907d6572d367e48e97d90c83ca5/CMakeLists.txt#L299-L303
But I'm not sure if we actually need to replicate that here. It only affects older CMake versions.